pages/code-of-conduct.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: page
+title: "Code of Conduct"
+permalink: "/code-of-conduct/"
+header:
+---
+
+
+
+## Diversity Statement
+
+The INTERSECT project welcomes and encourages participation in our community by people of all backgrounds and identities. The INTERSECT project is committed to promoting and sustaining a culture that values mutual respect, tolerance, and learning. We work together as a community to help each other live out these values.
+
+We have created this diversity statement because we believe that a diverse community is stronger, more vibrant, and produces better
+software and better research. A diverse community where people treat each other with respect has more potential contributors,
+more sources for ideas, and fewer shared assumptions that might hinder the achievement of our goals.
+
+Although we have phrased the formal diversity statement generically to make it all-inclusive, we recognize that there are specific
+identities that are impacted by systemic discrimination and marginalization. We welcome all people to participate in INTERSECT activities regardless of their identity or background.
+
+## Introduction & Scope
+
+This code of conduct should be honored by everyone who participates in INTERSECT activities. This code of conduct applies to all spaces managed by the INTERSECT project, including all public and private mailing lists, issue
+trackers, wikis, forums, code repositories, and any other communication channel used by our community. The code of conduct equally
+applies at INTERSECT events and governs standards of behavior for attendees, speakers, volunteers, booth staff, and event
```

Comment:
```suggestion
applies at INTERSECT events and governs standards of behavior for attendees, participants, speakers, instructors, volunteers, and event
```

Reply:
I don't think we need booth staff, but I added participants and instructors to be more specific to our types of activities.